src/core/coap/coap_secure.hpp

```diff
@@ -106,8 +125,9 @@ class CoapSecure : public CoapBase
     /**
      * This method initializes DTLS session with a peer.
      *
-     * @param[in]  aMessageInfo  A reference to an address of the peer.
-     * @param[in]  aCallback     A pointer to a function that will be called once DTLS connection is established.
+     * @param[in]  aMessageInfo            A reference to an address of the peer.
```

Comment:
nit: update the description for `aMessageInfo`?

Reply:
Can i use the description from udp `A pointer to a message info structure.` or would you like it more specific?